docs/src/modules/sandbox/StackBlitz.ts

```diff
@@ -64,8 +65,9 @@ export default defineConfig({
     'index.html': CRA.getHtml({ ...demoData, main: `/src/index.${ext}` }),
     'package.json': JSON.stringify(
       {
-        name: 'mui-demo',
+        version: process.env.LIB_VERSION,
```

Comment:
Because we set "latest" for the version, and most people who create reproduction never pin the version, having the versions set at the root should help us reproduce 2+ years old bugs when we try to iterate on some logic.

Reply:
That's not very useful to recreate the installed environment. First, our packages follow independent versioning so it can't be captured in a single version number. And second, it won't capture transitive dependencies. i.e. you may fix the @mui/material version to 7.3.6, its @mui/utils dependency is defined with caret range, so it may as well install 7.56.12 of that package.
There will be a lockfile in the sandbox of the reproduction which should be sufficient to recreate and inspect the conditions of the bug.

Reply:
> That's not very useful to recreate the installed environment

@Janpot Oh yeah, agree, it only helps to know it's v4 and not v5 or v6 😆. 

> There will be a lockfile in the sandbox of the reproduction which should be sufficient to recreate and inspect the conditions of the bug.

In the reproduction provided in https://github.com/mui/material-ui/issues, none seems to have this.

(_A side note, they all seem to prefer CodeSandbox even when we push to StackBlitz by default in the instructions, maybe some SEO lag, or something else._)

Reply:
> it only helps to know it's v4 and not v5 or v6

Wouldn't it make more sense then to just replace "latest" with "<current-major>"?

Reply:
Yeah, I think it would be better, but this is more time-consuming to implement. I think we can narrow the scope down, make the PR simpler.